.github/workflows/tests.yml

```diff
@@ -215,7 +215,7 @@ jobs:
       - name: run tests
         run: |
           cd build
-          ctest --build-config ${{ matrix.build-type }} --parallel 2
+          ctest --output-on-failure --build-config ${{ matrix.build-type }} --parallel 2
```

Comment:
Do you intend to keep "--output-on-failure"?

Reply:
Yes, this make debugging from CI much nicer